### DISCUSSION TITLE
merge problem

### DISCUSSION THREAD

**Discussion by mrdiep999** [community/community#65910] [Programming Help] [answered]
### Body

how to merge your file github without conflic

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Linh777GKOG** [comment, marked as answer]
Just pull the Latest Changes from the Remote Repository => Make Your Local Changes => Commit Your Changes => Push Your Changes to GitHub => Pull Any New Changes (Optional) => Resolve Conflicts => Push Again

**Comment by thanmaisai** [comment]
To merge a file on GitHub without conflict, pull the latest changes, make your changes, commit and push, then pull again and resolve any conflicts.

Pull | Get the latest changes from the remote repository.
-- | --
Make changes | Make your changes to the file.
Commit | Save your changes to your local repository.
Push | Share your changes with the remote repository.
Pull again | Get any new changes that have been made to the remote repository since you last pulled.
Resolve conflicts | If there are any conflicts, manually resolve them before pushing your changes again.
